### ISSUE TITLE
Parameterize the date formats.

### ISSUE THREAD

**Issue by Albejr** [chase0213/ng-infinite-calendar#7]
#### Proposal:
>Parameterize the date formats.

#### What is the expected behavior?

>01/2019
> Jan-2019
>Janeiro-2019

#### What is the current behavior?

```
<span class="current-day-indicator">
    {{ midDayOnCurrentWindow.strftime("%Y/%m") }}
</span>
```

#### What are the steps to reproduce?
![image](https://user-images.githubusercontent.com/24717256/51476248-1ce5b380-1d6c-11e9-97ec-c1063441dc39.png)

#### What is the use-case or motivation for changing an existing behavior?
>Multi languages.

#### Which versions of package, node, npm or other packages are affected?
>0.1.4

#### Is there anything else we should know?
```
ptBR: {
    dayOfWeek: {
        default: ['Domingo', 'Segunda-Feira', 'Terça-Feira', 'Quarta-Feira', 'Quinta-Feira', 'Sexta-Feira', 'Sábado'],
        short: ['Dom', 'Seg', 'Ter', 'Qua', 'Qui', 'Sex', 'Sab']
    },
    months: {
        default: ['Janeiro', 'Fevereiro', 'Março', 'Abril', 'Maio', 'Junho', 'Julho', 'Agosto', 'Setembro', 'Outubro', 'Novembro', 'Dezembro'],
        short: ['Jan', 'Fev', 'Mar', 'Abr', 'Mai', 'Jun', 'Jul', 'Ago', 'Set', 'Out', 'Nov', 'Dez'],
    }
}
```

**Comment by chase0213** [comment]
@Albejr Good issue!
Do you want to create a PR?
If you don't, I will implement something like the following:

**i18n.ts**

```js
"enUS": {
  "dateFormat": {
    "default": "%d/%m/%y",
    "short": "%m/%y"
  },
  "months": {
    "default": ["January", ...],
    "short": ["Jan", ...]
  }
}
```

**Comment by Albejr** [comment]
I await the next version.
ありがとう

**Comment by Albejr** [comment]
like this:
[https://angular.io/api/common/DatePipe](https://angular.io/api/common/DatePipe)

```
<span class="current-day-indicator">
    {{ midDayOnCurrentWindow | date: dateFormat }}
</span>
```

ありがとう

**Comment by Albejr** [comment]
I need to show the month in the format name, rather than the number.
How to do this in v.0.1.5?

![image](https://user-images.githubusercontent.com/24717256/51620604-e352ab00-1f19-11e9-8919-1d6379b6d617.png)

**Comment by chase0213** [comment]
@Albejr 
Sorry, v0.1.5 doesn't support the format.

But I've just released v0.1.6, which supports what you want to do!
You can use any formats [angular's date pipe supports](https://angular.io/api/common/DatePipe), now.

In your case, do like this:

```html
<infinite-calendar [monthDateFormat]="'MMM/yyyy'"></infinite-calendar>
```

The attributes `dateFormat` and `monthDateFormat` overwrites default settings.
If and only if you do not pass these attributes, the default language settings are used.

Thank you for your contribution!

**Comment by Albejr** [comment]
Very good. TKS
